source/PlayState.hx

```diff
@@ -39,12 +43,21 @@ class PlayState extends FlxState
 			1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],
 			20, 15, AssetPaths.tiles__png, 32, 32);
 		add(map);
+		
+		//make-a da blockies
+		blockGroup.add(new Block(3, 8, true));
+		blockGroup.add(new Block(4, 8, true));
+		blockGroup.add(new Block(7, 6));
+		blockGroup.add(new ItemBlock(8, 6, "Fake Item"));
+		blockGroup.add(new FallingBlock(9, 6));
+		add(blockGroup);
 	}
 
 	override public function update(elapsed:Float):Void
 	{
 		super.update(elapsed);
 		
 		FlxG.collide(map, player);
+		FlxG.collide(blockGroup, player, function(b:Block, p:Player) {b.onTouch();} );
```

Comment:
Just as a side note, if you want to execute your own collision logic without any of HaxeFlixel's default behavior, you can use the `FlxG.overlap()` method, which is pretty much the same, minus their built-in collision-resolving function.